src/zones/dbm.rs

```diff
@@ -676,6 +676,79 @@ impl DBM<Valid> {
         unsafe { dbm.assert_valid() }
     }
 
+    fn compute_tables(&self, src_clocks: &Vec<bool>, dst_clocks: &Vec<bool>) -> (Vec<ClockIndex>, Vec<ClockIndex>) {
+        let mut dst_to_src = Vec::<ClockIndex>::default();
+        let mut src_to_dst = Vec::<ClockIndex>::default();
+        let mut src_ind = 0;
+        assert_eq!(src_clocks.len(), dst_clocks.len());

```

Comment:
unhandled panic. intentional?

Reply:
Yes, if these two arent equal the program wouldn't behave as expected.